### DISCUSSION TITLE
Please add YouTube Playlist Support in Video or Playlist Widget

### DISCUSSION THREAD

**Discussion by starstudiomn** [elementor/elementor#27343] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I can't embed a YouTube playlist using one of the built in widgets, and get it to load the official YouTube embedded playlist.

### Describe the solution you'd like.

Preferably the video widget would accept playlist URLs.  The official embed feature looks almost identical to a regular video embed, but it automatically loads new videos added.  I know I can do it with HTML, but it's just a bit more work not being able to customize it right in the Elementor interface.

### Describe alternatives you've considered.

It can be solved by going to YouTube, customizing the embed widget, and pasting in the HTML code.  But this makes later customization more difficult.

### Additional context

Here's what I'd like to appear when I paste a playlist URL.  You can tell it's different from a regular video by the icon in the top right.  But otherwise it's nearly identical.  Should be pretty quick to add parsing of playlist URLs.

<img width="1728" alt="Screenshot 2024-05-15 at 6 25 41 PM" src="https://github.com/elementor/elementor/assets/166424561/9b4ba2de-9e17-46f8-9614-56d2044060b6">

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by IsotonikStudios** [comment]
This is so essential!!
